src/plugins/data-card-tool/components/case-attribute.tsx

```diff
@@ -110,6 +111,10 @@ export const CaseAttribute: React.FC<IProps> = observer(props => {
     setCurrEditAttrId(attrKey);
   };
 
+  const handleDeleteAttribute = () => {
+    // confirmation dialogue
```

Comment:
```suggestion
    // confirmation dialog
```
"Dialogue" is a conversation. "Dialog" is a UI element.